plugins/lightspeed-backend/src/service/router.ts

```diff
@@ -24,25 +34,57 @@ export async function createRouter(
     response.json({ status: 'ok' });
   });
 
+  router.get(
+    '/session/load',
+    validateLoadHistoryRequest,
+    async (request, response) => {
+      const { conversation_id, historyLength }: SessionLoadRequestBody =
+        request.body;
```

Comment:
I think instead of using query params for conversation ID we can use it as part of the path parameter. The main param like conversation ID could be part of the path param and others could be query param for `GET` requests. For `DELETE` we only need conversation id so it would be in path param. Using query params for DELETE is not considered a good `REST` practice. 

So  we could have APIs like this - 

`GET` - `/session/:conversation_id?historyLength=30`
`DELETE` - `/session/:conversation_id`

Reply:
as discussed, we decided:
```
GET         /conversations                       -  list all conversations
GET         /conversations/:conversation_id      - retrieve a conversation details
POST       /conversations                                       - creates a new conversation
DELETE   /conversations/:conversation_id     - delete a conversatio
```
I've updated this PR according. 
This PR implements the `GET/DELETE   conversations/:conversation_id `, PTAL again. @rohitkrai03 @karthikjeeyar